.changeset/famous-rice-taste.md

```diff
@@ -0,0 +1,6 @@
+---
+"@gradio/upload": patch
+"gradio": patch
+---
+
+fix:Live audio streaming output
```

Comment:
Let's include a highlight here, see https://github.com/gradio-app/gradio/pull/5057 for an example